### ISSUE TITLE
Adjusting spot size and control points independently/symmetrically

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#6335]
RawPedia states,

> The shapes have four control points, which can be adjusted independently or symmetrically.

It states that symmetric/independent adjustments are possible, but it does not explain _how_ to do it. I tried the obvious - ctrl, alt, shift, and combinations of the three - while dragging control points, but nothing worked. It also doesn't explain how to adjust spot size.

I am aware of the hidden "Shape method" setting in the puzzling "Specific cases" menu, but would expect symmetric/independent adjustment and spot size adjustment to work the same way it does in many other programs - on-canvas using the mouse and a modifier key. If this was implemented then we could get rid of that "Shape method" combo and all of its associated widgets, and also the "spot-size" slider. Remember that the only reason the "slider" methods are there is to handle control points that are not visible because they're off-canvas (ref. https://github.com/Beep6581/RawTherapee/issues/2538#issuecomment-130048857 ), and that is no longer a valid reason since in the current code `5.8-3022-g`b1e7860a2 the control points are clearly visible when they're off-canvas.

![screenshot_20210813_144429](https://user-images.githubusercontent.com/5844619/129358961-1427538c-7244-4003-9d9c-2b54cc979499.png)

For reference, RawTherapee already allows on-canvas spot size adjustment in the "Spot Removal" tool.

So:
1. I will rename "Shape method" to "Spot adjustment".
2. @Desmis is the only way of making a control point adjustment symmetric/independent via the combobox, and the only way of adjusting the spot size via the slider? I will update RawPedia accordingly once confirmed.
3. I'm making a feature request to enable independent/symmetric adjustment via on-canvas adjustments, same for spot size, and getting rid of that entire "Shape method" combo and associated widgets.

**Comment by Stuntflyer** [comment]
This will be a welcome addition. Looking forward to it being implemented.

**Comment by waynesutton50** [comment]
>     2\. the only way of adjusting the spot size via the slider

The "Spot size" slider label is slightly ambiguous because it refers to the variable-diameter circle inside the RT-spot and not to the area delimited by the control points. Perhaps it would be better to change the label to something like "Spot circle size"? 
EDIT: or alternatively "Spot center size"

**Comment by Desmis** [comment]
@Beep6581 
Yes, this is the only way making a control point adjustment symmetric/independent via the combobox, and the only way of adjusting the spot size via the slider.
And for the rest (labels, GUI, Rawpedia,..) do what you want I'm not at all  a GUI specialist, nor RT owner.

jacques
